### ISSUE TITLE
Update NativeBase react-native-vector-icons dependency to 0.4.0. Currently breaks for react-native 0.40

### ISSUE THREAD

**Issue by Monte9** [GeekyAnts/NativeBase#421]
Started a new React Native project and I am unable to even run a simple example using NativeBase. 

![screen shot 2017-01-10 at 7 12 47 am](https://cloud.githubusercontent.com/assets/7840686/21790718/3a746bea-d704-11e6-8e1e-6a9092fd703a.png)

Here are my dependencies:
```
"native-base": "^0.5.20",
"react": "~15.4.0-rc.4",
"react-native": "0.40.0"
```
This is a known issue and has been reported on `react-native-vector-icons` repo ([HERE](https://github.com/oblador/react-native-vector-icons/issues/373)). Turns out the new release 0.4.0 fixes this problem. 

Although since NativeBase is internally using `react-native-vector-icons` 0.3.0 I am not sure how to update the version myself. 

https://github.com/GeekyAnts/NativeBase/blob/master/package.json#L27

Any suggestions on a workaround? If not, when will the dependency be updated? Thanks.

Also found a PR that is related to this: https://github.com/GeekyAnts/NativeBase/pull/414
Consider merging it soon?

**Comment by stonepreston** [comment]
I installed the icons as a dependency of my project and it seems to work

`npm install --save react-native-vector-icons

**Comment by AWDGroup** [comment]
+1 same issue

**Comment by sankhadeeproy007** [comment]
Has been updated in master. Please use from master till the moment till we release a newer version.

**Comment by sankhadeeproy007** [comment]
Fixed in 0.5.21
